src/components/UriViewer/UriViewer.js

```diff
@@ -111,22 +188,7 @@ export const UriViewer = _.flow(
           }, ['View this file in the Google Cloud Storage Browser'])
         ]),
         h(UriDownloadButton, { uri, metadata, accessUrl }),
```

Comment:
In other cases, the workspace is not passed to `UriDownloadButton`. Can `workspace` not be passed to `UriDownloadButton` here as well? That would eliminate the need to branch in `UriDownloadButton`.

Reply:
it definitely can, I think I was overcomplicating this since I thought azure and gcp behavior had to be different.